iceoryx_posh/include/iceoryx_posh/internal/runtime/ipc_interface_base.hpp

```diff
@@ -23,20 +23,20 @@
 #include "iceoryx_hoofs/internal/posix_wrapper/unix_domain_socket.hpp"
 #include "iceoryx_hoofs/internal/relocatable_pointer/relative_pointer.hpp"
 #include "iceoryx_hoofs/internal/units/duration.hpp"
+#include "iceoryx_hoofs/platform/errno.hpp"
 #include "iceoryx_hoofs/platform/fcntl.hpp"
 #include "iceoryx_hoofs/platform/stat.hpp"
+#include "iceoryx_hoofs/platform/time.hpp"
 #include "iceoryx_hoofs/platform/types.hpp"
 #include "iceoryx_hoofs/platform/unistd.hpp"
 #include "iceoryx_hoofs/posix_wrapper/named_pipe.hpp"
 #include "iceoryx_posh/iceoryx_posh_types.hpp"
 #include "iceoryx_posh/internal/runtime/ipc_message.hpp"
 
 #include <cstdint>
-#include <errno.h>
 #include <stdio.h>
 #include <stdlib.h>
```

Comment:
Please write here `#include <cstdlib>` instead of `stdlib.h`.

Reply:
Is it actually possible with your script to check that no libc headers are included in c++ files but instead the c++ version of the c header?
Like with `cstdio` instead of `stdio.h` since it is best practices.

Reply:
Good point, I fixed it manually. And added the check for the C++ version as an item to the issue.